pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -171,13 +171,17 @@ func (i *CommandInitOption) Complete() error {
 	}
 
 	// Determine whether KarmadaDataPath exists, if so, delete it
+	// Related issue: https://github.com/karmada-io/karmada/pull/1455
+	// When the current host uses init to install karmada multiple times, the installation fails
+	// because the data directory of karmada is not empty.
 	if utils.IsExist(i.KarmadaDataPath) {
 		if err := os.RemoveAll(i.KarmadaDataPath); err != nil {
 			return err
 		}
 	}
 
-	return nil
+	// Create this dir, next steps will download crds to this dir
```

Comment:
We can add comments here.

Reply:
U mean: add comment like `TODO: enhence this operation like kubeadm reset`?

Reply:
No. The comment could be shorter and clearer. Like above comments

```
// When the current host uses init to install karmada multiple times, the installation fails
// because the data directory of karmada is not empty.
```

It is not necessary to describe the `install karmada multiple times` that should be in PR description rather than in code.